highcharts_core/global_options/shared_options.py

```diff
@@ -42,7 +44,7 @@ def to_js_literal(self,
 
         as_str = prefix + options_body + ');'
 
-        if filename:
+        if filename and validate.path(filename):
```

Comment:
Can be simplified to:
```suggestion
        if validators.path(filename, allow_empty = True):
```